docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -252,6 +252,69 @@ Here's what you need to know about upgrading:
 
 >TIP: For a smooth transition, create the new stack first. After the new stack is bootstrapped and ready to serve, delete the old stack. If you are using a load balancer, first get the new instances up and running and then convert the DNS name from the previous one to the new one.
 
+## Logs
+### Where to read logs
+To access the logs, the participants need to ssh into the EC2 instance and access the logs.
+
+The logs are located at `/var/logs/` . The logs are in the format of `operator.log-<timestamp rotated>` . 
+
+### Default log settings
+#### logrotate config
+- After deploying the operator instance, the default log rotation settings will be applied:
+    - 30 rotations of logs will be kept
+    - Normally if no abnormal logs, it should be equivalent to 30 days' worth of logs as it will be rotated daily
```

Comment:
Suggest:
- If there are no abnormal log entries, logs are rotated daily, so log history is equivalent to 30 days of data.

Also this made me wonder what would happen if it's abnormal. That's answered in the next point but, actually, it's just if it's abnormally large. So perhaps we could say that:
- If log entries are not abnormally large, (etc)